src/components/app.js

```diff
@@ -19,51 +20,88 @@ import NewIssue from 'components/issues/new-issue/NewIssue';
 
 import {
   getUserInfoWithToken,
-  saveTokenToLocalStorage
+  saveTokenToLocalStorage,
+  getUserTokenWithCode
 } from 'actions/user.actions';
 
 type ConnectedProps = {
   getUserInfoWithToken: () => void,
-  saveTokenToLocalStorage: () => void
+  saveTokenToLocalStorage: () => void,
+  isAuthenticated: string | null
 };
 
 type OwnProps = {};
 
 class App extends React.Component<ConnectedProps & OwnProps> {
   componentWillMount() {
     const user = get('user', loadFromStorage('auth'));
```

Comment:
Loading directly from localStorage in a component is a bad practice, it is better handled in the redux layer using something like redux-persist.

So if the user has logged in and the reducer was updated that value will stay for the next page loads (and we won't need to load it directly ourselves).

Reply:
In addition please do not use componentWillMount as it is deprecated and will be removed in the next react releases